docs/rule/no-whitespace-for-layout.md

```diff
@@ -0,0 +1,42 @@
+# no-whitespace-for-layout
```

Comment:
Would you be able to add some valid / invalid code samples in the documentation? Specifically, it would help to suggest some alternatives (good examples) for linebreaks such as `<br>`, CSS padding/margin, etc.

There's a `_TEMPLATE_.md` with a suggested documentation outline including the good/bad example section.

Reply:
updated, sorry 'bout that.